### PR TITLE
TATB - 01 :#BUG : Make login and registration page responsive till tablet view

### PR DESCRIPTION
**Pull request by PrabhatJTDeveloper**
- for making login and registration form responsive change html and scss file of the both component.
     - remove the global font-family scss from all component and give it to the style.scss

### CODE REVIEW

**Review by charmijtdev — APPROVED** [review]

File: src/app/authorization/login/login.component.html [hunk 1]
```diff
@@ -1,63 +1,92 @@
...
+              Email is required
+            </div>
+
+            <div
+              class="form-text text-danger"
+              *ngIf="(loginFormControl['emailId']?.errors?.['email'] && (loginFormControl['emailId']?.dirty || loginFormControl['emailId']?.touched))"
+            >
+            Email is Invalid
+            </div>
+          </div>
```
Comment (PoojanPatelJT) [inline src/app/authorization/login/login.component.html:37]: Please remove multiple dive for showing error message merge it inside one div

File: src/app/authorization/login/login.component.html [hunk 2]
```diff
@@ -1,63 +1,92 @@
...
+              *ngIf="(loginFormControl['password']?.errors?.['minlength'] || loginFormControl['password']?.errors?.['maxlength'] && (loginFormControl['password']?.dirty || loginFormControl['password']?.touched))"
+            >
+              Password should be at least 6 and at the max 12 characters long
+            </div>
           </div>
-          <div class="form-text text-danger" *ngIf="">
-            Password should be at least 6 and at the max 15 characters long.
+        </div>
+
+        <div class="row p-3">
```
Comment (PoojanPatelJT) [inline src/app/authorization/login/login.component.html:64]: Please merge error div here also

File: src/app/authorization/login/login.component.scss [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
-    height: auto;    
-}
\ No newline at end of file
+.image {
+  width: 4rem;
+  height: 4rem;
+}
+label {
+  font-size: calc(70% + 1vh);
+}
```
Comment (PoojanPatelJT) [inline src/app/authorization/login/login.component.scss:7]: please use class name not HTML tag directly until unless it is necessary

File: src/app/authorization/login/login.component.html (hunk 2 above)
Comment (charmijtdev) [reply src/app/authorization/login/login.component.html:64]: This change is in upcoming PR

File: src/app/authorization/login/login.component.scss (hunk 3 above)
Comment (charmijtdev) [reply src/app/authorization/login/login.component.scss:7]: This change is in upcoming PR

File: src/app/authorization/login/login.component.ts [hunk 4]
```diff
@@ -40,32 +40,33 @@ export class LoginComponent implements OnInit, OnDestroy {
...
+            this.loginForm?.reset();
           }
         });
     }
   }
 
   ngOnInit(): void {
+    this.createForm();
     if (!localStorage.getItem('registeruser')?.length) {
       let registeruser: any = localStorage.getItem('registeruser');
```
Comment (PoojanPatelJT) [inline src/app/authorization/login/login.component.ts]: Var should have camelCase

**Review by PoojanPatelJT — APPROVED** [review]

**Comment by charmijtdev** [comment]
This PR is approved and merged

**Pull request merged**